test/e2e/performanceprofile/functests/13_llc/llc.go

```diff
@@ -486,73 +490,348 @@ var _ = Describe("[rfe_id:77446] LLC-aware cpu pinning", Label(string(label.Open
+				Entry("[test-id:81671] With multiple guaranteed containers where total cpus requested is less than single ccx size", "test-deployment3", "PartialAlignment", []string{"test2"}),
+			)
+		})
+	})
+
+	Context("Functional Tests with SMT Disabled", func() {
+		var (
+			L3CacheGroupSize   int
+			totalOnlineCpus    cpuset.CPUSet
+			nosmt              map[int][]int
+			getCCX             func(cpuid int) (cpuset.CPUSet, error)
+			reserved, isolated cpuset.CPUSet
+			policy             = "single-numa-node"
+		)
+		BeforeAll(func() {
+			profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+			Expect(err).ToNot(HaveOccurred())
+			ctx := context.Background()
+			for _, cnfnode := range workerRTNodes {
+				numaInfo, err := nodes.GetNumaNodes(ctx, &cnfnode)
+				Expect(err).ToNot(HaveOccurred())
+				coresiblings, err := nodes.GetCoreSiblings(ctx, &cnfnode)
+				Expect(err).ToNot(HaveOccurred(), "Unable to get numa information from the node")
+				nosmt = transformToNoSMT(coresiblings)
+				if len(numaInfo) < 2 {
+					Skip(fmt.Sprintf("This test need 2 Numa nodes. The number of numa nodes on node %s < 2", cnfnode.Name))
 				}
-				podLabel := make(map[string]string)
-				podLabel["test-app"] = fmt.Sprintf("telcoApp-%d", i)
-				testlog.TaggedInfof("Deployment", "Creating Deployment %v", deploymentName)
-				dp, err := createDeployment(ctx, deploymentName, podLabel, &targetNode, rl)
 				Expect(err).ToNot(HaveOccurred())
-				dpList = append(dpList, dp)
 			}
-			defer func() {
-				// delete deployment
-				for _, dp := range dpList {
-					testlog.TaggedInfof("Deployment", "Deleting Deployment %v", dp.Name)
-					key := client.ObjectKeyFromObject(dp)
-					err := testclient.Client.Get(ctx, key, dp)
-					Expect(err).ToNot(HaveOccurred(), "Unable to fetch podlabels")
-					podLabels := dp.Spec.Template.ObjectMeta.Labels
-					err = testclient.DataPlaneClient.Delete(ctx, dp)
-					Expect(err).ToNot(HaveOccurred(), "Unable to delete deployment %v", dp.Name)
-					waitForDeploymentPodsDeletion(ctx, &targetNode, podLabels)
+			node0 := cpuset.New(nosmt[0]...)
+			node1 := cpuset.New(nosmt[1]...)
+			reserved = cpuset.New(nosmt[0][:4]...)
+			node0remaining := node0.Difference(reserved)
+			isolated = (node0remaining.Union(node1))
+			// Modify the profile such that we give 1 whole ccx to reserved cpus
+			By("Modifying the profile")
+			reservedSet := performancev2.CPUSet(reserved.String())
+			isolatedSet := performancev2.CPUSet(isolated.String())
+			testlog.Infof("Modifying profile with reserved cpus %s and isolated cpus %s", reserved.String(), isolated.String())
+			profile.Spec.CPU = &performancev2.CPU{
+				Reserved: &reservedSet,
+				Isolated: &isolatedSet,
+			}
+			profile.Spec.NUMA = &performancev2.NUMA{
+				TopologyPolicy: &policy,
+			}
+
+			profile.Spec.AdditionalKernelArgs = []string{"nosmt", "module_blacklist=irdma"}
```

Comment:
where do we revert this setting?

Reply:
We have a AfteraAll section at the start. 
https://github.com/openshift/cluster-node-tuning-operator/blob/main/test/e2e/performanceprofile/functests/13_llc/llc.go#L130